src/Core/OrganizationFeatures/OrganizationSponsorships/FamiliesForEnterprise/Cloud/CloudSyncSponsorshipsCommand.cs

```diff
@@ -54,11 +54,13 @@ private async Task<(IEnumerable<OrganizationSponsorshipData> data, IEnumerable<O
             var sponsorshipIdsToDelete = new List<Guid>();
             var sponsorshipsToReturn = new List<OrganizationSponsorshipData>();
 
+            var sponsoringOrganizationProduct = StaticStore.GetPlan(sponsoringOrg.PlanType).Product;
+            var sponsoringOrgDisabledForMoreThanGracePeriod = OrgDisabledForMoreThanGracePeriod(sponsoringOrg);
+
```

Comment:
These both have very little value, they're just lookups on objects in memory and a little calculation. I'm not sure it's worth the extra lines needed to store this

Reply:
Alright, reverted.